unlock-protocol.com/blog/version-1-2.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Smart Contract Upgrade: Version 1.2"
+subTitle: We shipped a large upgrade to our Unlock contract and the new locks have new features!
+authorName: Julien Genestoux
+publishDate: January 9, 2020
+description: We wanted to start the year with a big release, so we shipped a very large upgrade (the largest since June 2018) to out smart contracts. This new version brings a ton of new features which will venetually bubble up to the Unlock user interfaces (and beyond!).
+image: /static/images/blog/unlocking-smart-contracts/code.jpeg
+---
+
+We wanted to start the year with a big release, so we shipped a very large upgrade (the largest since June 2018) to out smart contracts. This new version brings a ton of new features which will eventually bubble up to the Unlock user interfaces (and beyond!).
+
+# The Unlock Contract
+
+First, a quick reminder about the architecture behind our smart contracts. First, there is the `Unlock` contract. It's a factory contract, whose main job is to deploy locks for creators. This contract is _currently_ owned by us, at Unlock Inc, through the use of a multi-sig contract (powered by [Gnosis](https://gnosis.io/)), but our hope is that eventually, it can be co-owned by the whole Unlock community.
```

Comment:
Minor: Starting two sentences in a row with `First, `... maybe remove the first first for `A quick reminder... First, there is`